src/services/chat/prepopulated.js

```diff
@@ -41,17 +41,17 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser
+                  ...ownUser.defaulValues
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the property name `defaulValues`. It should be `defaultValues` if it exists on the `ownUser` object.
```diff
- ...ownUser.defaulValues
+ ...ownUser.defaultValues
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->